### PR TITLE
extend test setup and make it work with scorpio

### PR DESCRIPTION
**Pull request by wistefan**

### CODE REVIEW

**Review by beknazaresenbek — APPROVED** [review]

**Review by dwendland — APPROVED** [review]

File: logs
Comment (pulledtim) [inline logs:1]: Pls remove

File: k3s/scorpio/scorpio.yaml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    spec:
+      serviceAccountName: default
+      containers:
+        - name: scorpio
+          imagePullPolicy: IfNotPresent
+          image: "scorpiobroker/all-in-one-runner:java-kafka-4.1.4"
+          env:
+            - name: DBHOST
+              value: postgis
+            - name: BUSHOST
```
Comment (pulledtim) [inline k3s/scorpio/scorpio.yaml:115]: Is that all thats necessary to integrate Kafka too? Nice :D

**Review by pulledtim — APPROVED** [review]

File: k3s/scorpio/scorpio.yaml (hunk 1 above)
Comment (wistefan) [reply k3s/scorpio/scorpio.yaml:115]: In scorpio yes, but the Kafka deployment itself is in the other yaml file;)

Comment (pulledtim) [reply k3s/scorpio/scorpio.yaml:115]: Yea I saw that, lovely slim and easy deployment :sweat_smile:

Comment (pulledtim) [reply k3s/scorpio/scorpio.yaml:115]: Will scorpio read from Kafka too?

Comment (wistefan) [reply k3s/scorpio/scorpio.yaml:115]: Yes. Scorpio (in contrast to Orion-LD) is built in a microservice architecture and uses kafka as the bus(similar to the kiwi-bus;)) between the individual services. Its normally not only one container, but they also publish the "aio" container, that holds all of the services in a single container. I used this one, to keep the setup more readable.

Comment (pulledtim) [reply k3s/scorpio/scorpio.yaml:115]: kiwi-bus :face_exhaling: Not making this better :laughing: 
How are Kafka and the Blockchain going to work together? "uncritical" entities on kafka and orders&co on blockchain?

Comment (wistefan) [reply k3s/scorpio/scorpio.yaml:115]: There is no blockchain here. The kafka is just for inter-service communication inside scorpio. The blockchain in dome is completely seperated from that.

**Pull request merged**